### PR TITLE
Split Getting Started Guides for Kubernetes/Swarm

### PR DESCRIPTION
**Pull request by carolynvs**
Pointing at master in order to get previews. I will repoint at dev before merging.

- [x] Add redirects for any updated slugs (see https://github.com/getcarina/carina-deconst-control/pull/2)
- [x] Update existing links (Left blog post links alone, redirects will take care of them)

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-1aadcbb091/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-fde8759608/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-b08ff8e128/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-aeb1936994/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-5bbb3d03df/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-d488475904/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-7df87b09cc/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-e70929dba7/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-3a38e68ccd/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-1f259b99d5/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-369e2f3c85/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-369e2f3c85/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-d893730c3b/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-d893730c3b/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-1bdf0d6089/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-ebc38379c0/)

File: _includes/getting-started-with-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+### Run your first application on Kubernetes
```
Comment (jamiehannaford) [inline _includes/getting-started-with-kubernetes.md]: I didn't know you were working on this! I saw the card unassigned so I wrote the "Zero to Hero" guide yesterday, which deals with setting up a WordPress/MySQL installation. The article I wrote though goes into detail about:

- Services
- Deployments
- PersistentVolumes
- Secrets
- PersistentVolume claims
- Ingress

So maybe we can have 2 articles, one which is an introduction to setting up WP in under 5 mins. And another which is a zero-to-hero guide which discusses all the main Kubernetes concepts and goes more in-depth (maybe a 30 mins tutorial). What do you think? I'm cautious about possible duplication, but considering the articles have different aims maybe we'll be okay

Comment (jamiehannaford) [inline _includes/getting-started-with-kubernetes.md]: I've submitted a PR for the zero-to-hero guide: https://github.com/getcarina/getcarina.com/pull/889. Needs testing first, but would love to hear your feedback on it too :)

Comment (carolynvs) [inline _includes/getting-started-with-kubernetes.md]: Agreed! It really does need to be two separate articles. One that someone can just follow to say "I setup a thing!" and another that explains all the steps and components in detail.

File: _getting-started/2015-10-26-overview-of-carina.md [hunk 2]
```diff
@@ -90,4 +90,4 @@ Carina has a CLI that you can use to control clusters from a terminal or scripts
 
 ### Next step
 
-Run your first containerized application by [getting started on Carina]({{ site.baseurl }}/docs/getting-started/getting-started-on-carina/).
+Run your first containerized application by [Getting started with Docker Swarm]({{ site.baseurl }}/docs/getting-started/create-swarm-cluster/) or [Getting started with Kubernetes]({{ site.baseurl }}/docs/getting-started/create-kubernetes-cluster/).
```
Comment (jamiehannaford) [inline _getting-started/2015-10-26-overview-of-carina.md]: Should we also include links to K8s?

Comment (carolynvs) [inline _getting-started/2015-10-26-overview-of-carina.md]: What would you like to link to? General K8's doc or one of the new k8's pages?

For new content, I've been going by the rule of thumb that I don't link to pages that doesn't exist in the main branch yet, so if a PR adds a cool new guide, that PR also adds links to itself as needed.

Comment (jamiehannaford) [inline _getting-started/2015-10-26-overview-of-carina.md]: Sorry, I didn't see the second link in this sentence. I think it's fine as it is

**Comment by kmsholcomb** [comment]
Hi @carolynvs. I edited the four "getting started" articles; the marked-up Word docs (and PDF copies of them) are attached. My edits are pretty minor; you did a great job of breaking out all the information! Please let me know if you have any questions. Thanks!

[Getting started with Kubernetes and the Carina CLI_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/579297/Getting.started.with.Kubernetes.and.the.Carina.CLI_editsKH.pdf)
[Getting started with Kubernetes and the Carina CLI_editsKH.docx](https://github.com/getcarina/getcarina.com/files/579296/Getting.started.with.Kubernetes.and.the.Carina.CLI_editsKH.docx)
[Getting started with Kubernetes_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/579299/Getting.started.with.Kubernetes_editsKH.pdf)
[Getting started with Kubernetes_editsKH.docx](https://github.com/getcarina/getcarina.com/files/579298/Getting.started.with.Kubernetes_editsKH.docx)
[Getting started with Docker Swarm and the Carina CLI_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/579301/Getting.started.with.Docker.Swarm.and.the.Carina.CLI_editsKH.pdf)
[Getting started with Docker Swarm and the Carina CLI_editsKH.docx](https://github.com/getcarina/getcarina.com/files/579300/Getting.started.with.Docker.Swarm.and.the.Carina.CLI_editsKH.docx)
[Getting started with Docker Swarm_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/579302/Getting.started.with.Docker.Swarm_editsKH.pdf)
[Getting started with Docker Swarm_editsKH.docx](https://github.com/getcarina/getcarina.com/files/579303/Getting.started.with.Docker.Swarm_editsKH.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-1054871546/)

**Pull request merged**